bin/README.md

```diff
@@ -0,0 +1,3 @@
+# bin
```

Comment:
The only reason for this file's existence is to make sure the `bin/` dir is properly created everywhere. But that's already achieved with the `.gitignore` file. We can discard this one.